author/ieee/topics/markup.adoc

```diff
@@ -0,0 +1,158 @@
+---
+layout: ieee-flavor
+---
+
+= Metanorma markup for IEEE
+
+== Clauses
+
+=== Prefatory
+
+The abstract of the document is marked up as a clause with the style attribute `[abstract]`.
+
+The keywords of the document are given as the comma-delimited `:keywords:` document attribute.
+
+The introduction of the document is marked up as a clause with the title "Introduction".
+The initial boxed text, "This introduction is not part of P&lt;designation&gt;/D&lt;draft_number&gt;, Draft &lt;Gde./Rec. Prac./Std.&gt; for &lt;Complete Title Matching PAR&gt;." is automatically generated and populated by Metanorma.
+Do not provide it.
+
+The permissions of the document are marked up as a clause with the title "Acknowledgements".
+
+The templated material ("boilerplate") of the document front matter is all automatically provided by
+Metanorma:
+
+* Front page: Copyright statement, license statement
+* Second page: Contact statement, ISBNs
+* Important Notices and Disclaimers Concerning IEEE Standards Documents
+* Participants statement (the membership of the various committees is provided as document attributes)
+
+== Overview
+
+The _Overview_ clause, and its _Scope_, _Purpose_ and _Word Usage_ subclauses, are recognised automatically from the supplied clause headers.
+
+The _Word Usage_ subclause is mandatory, but is not autopopulated by Metanorma. The text is however included in
+the Metanorma template for IEEE.
+
+=== References
+
+The introductory paragraph for normative references and bibliographies is automatically generated by Metanorma.
+
+Normative references are automatically sorted by designator, by Metanorma.
+
+=== Definitions
+
+Definitions are recognised as a clause with the title "Definitions" or "Terms and definitions".
+
+Definitions are sorted by Metanorma.
+
+The notation for subdefinitions and cross-references in Metanorma is:
+
+[source,asciidoc]
+----
+=== widget
+
+alt:[doohickey]
+
+related:contrast[thing]
+
+related:see[thingummy]
+related:see[thingummijig]
+
+related:seealso[whatsit]
+
+related:equivalent[doovywhack] // Metanorma term for synonym
+
+[.definition]
+--
+device performing an unspecified function.
+--
+
+[.definition]
+--
+general metasyntactic variable.
+--
+----
+
+which renders as:
+
+____
+
+*widget*: *(A)* device performing an unspecified function. *(B)* general metasyntactic variable. _Contrast:_ thing.
+_Syn:_ doovywhack. _See:_ thingummijig; thingymmy. _See also:_ whatsit.
+____
+
+=== Annexes
+
+Normative and informative annexes are differentiated with the `normative` attribute:
+
+[source,asciidoc]
+----
+[appendix,normative=true]
+== Rules for implementation
+
+[appendix,normative=false]
```

Comment:
Is this normative syntax new? I thought we always used `obligation=normative`?

Reply:
Fixed